### PR TITLE
[RefManager] - Alpha Release Version

### PR DESCRIPTION
**Pull request by calvinhlee23**
### BE: 
- depends on https://github.com/ResearchHub/researchhub-backend/pull/1294

### Screenshot: 
Screenshot 2023-06-13 at 11 40 17 AM
Screenshot 2023-06-13 at 11 40 20 AM
Screenshot 2023-06-13 at 11 40 26 AM

### CODE REVIEW

File: components/ReferenceManager/references/ReferencesContainer.tsx [hunk 1]
```diff
@@ -232,6 +231,23 @@ function ReferencesContainer({
     setLoading(false);
   };
 
+  useEffectFetchOrgProjects({
```
Comment (yattias) [inline components/ReferenceManager/references/ReferencesContainer.tsx:234]: The functionality of these is good but as far as naming goes, I think we can omit the "Effect" and "Fetch" portions and just name: `useOrgProjects`.

This is common pattern in other packages and removes the need to know that the hook uses a "useEffect" or "fetch" behind the scenes. From the component using it's perspective, all that is needed to know is that the hook can be used to get projects somehow. 

Not a release blocker, just a suggestion.

File: components/ReferenceManager/references/reference_table/ReferencesTable.tsx [hunk 2]
```diff
@@ -131,17 +131,36 @@ export default function ReferencesTable({
...
+                  (item) => item.id === params?.row?.id
+                )
+              ),
+            });
             setIsDrawerOpen(true);
+          } else {
+            setSelectedReferenceIDs(params.id);
           }
         }}
+        // onCellDoubleClick={}
```
Comment (yattias) [inline components/ReferenceManager/references/reference_table/ReferencesTable.tsx:149]: Probably remove this?

File: config/api.js [hunk 3]
```diff
@@ -7,7 +7,8 @@ import { convertToBackendFilters } from "~/components/UnifiedDocFeed/utils/conve
 export const apiRoot = {
   production: "backend.researchhub.com",
   staging: "staging-backend.researchhub.com",
-  dev: "localhost:8000",
+  // dev: "localhost:8000",
+  dev: "127.0.0.1:8000" /* localhost */,
```
Comment (yattias) [inline config/api.js:11]: Can we keep localhost? There is some functionality that utilizes "localhost" as the HOST

**Review by yattias — APPROVED** [review]

File: components/ReferenceManager/references/ReferencesContainer.tsx (hunk 1 above)
Comment (calvinhlee23) [reply components/ReferenceManager/references/ReferencesContainer.tsx:234]: Will keep that in mind. Thanks for the tip!

**Pull request merged**